extensions/SemanticForms/includes/SF_ParserFunctions.php

```diff
@@ -212,7 +212,7 @@ static function renderQueryFormLink ( &$parser ) {
 		return $parser->insertStripItem( self::createFormLink( $parser, $params, 'queryformlink' ), $parser->mStripState );
 	}
 
-	static function renderFormInput( &$parser ) {
+	static function renderFormInput( $parser ) {
 		$params = func_get_args();
 		array_shift( $params ); // don't need the parser
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)